test/test_sedcli_basic.py

```diff
@@ -61,8 +81,76 @@ def test_initial_setup(self):
 		self.assertEqual(pipe.returncode, 0, "sedcli returned error while setting RLE and WLE on global range:\n" + str(errs))
 		self.assertEqual(len(errs), 0, "sedcli returned success code but error message has been printed on stderr:\n" + str(errs))
 
+		# Read Lock Disk

```

Comment:
Could we break this out into multiple test cases, e.g. 
1. test_initial_provision - test provisioning of the drive for security
2. test_lock - tests if locking works correctly that is this is locked and data can't be read or written,
Suggestion is to define smaller granularity tests allowing a person to run them individually instead of having one test testing everything.